jbmc/regression/jbmc/class-literals/test_lazy.desc

```diff
@@ -1,6 +1,6 @@
 CORE symex-driven-lazy-loading-expected-failure
 Test.class
---lazy-methods --function Test.main
+--function Test.main
```

Comment:
Will this test still fail (`symex-driven-lazy-loading-expected-failure`)? Applies for several others as well.